content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -340,7 +340,7 @@ startupProbe:
 ```
 
 Thanks to the startup probe, the application will have a maximum of 5 minutes
-(30 * 10 = 300s) to finish its startup.
+(30 \* 10 = 300s) to finish its startup.
```

Comment:
Was this meant to be a replacement of `\` instead of `*`? The original is correct. 30 failures, at 10 seconds per period, results in 300 seconds.

Reply:
Oh shoot, thanks for spotting that, i've fixed that back to the original